<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="DESKTOP-GTM5910" failures="1" tests="6" name="MBSTest" time="3.1" errors="0" timestamp="2024-02-20T13:55:44 IST">
  <testcase classname="practice.CW13_TestNgkey" name="test3" time="0.014"/>
  <testcase classname="practice.CW13_TestNgkey" name="test3" time="0.001"/>
  <testcase classname="practice.CW13_TestNgkey" name="test1" time="0.001"/>
  <testcase classname="practice.CW13_TestNgkey" name="test6" time="0.002"/>
  <testcase classname="practice.CW13_TestNgkey" name="test2" time="0.001"/>
  <testcase classname="practice.CW13_TestNgkey" name="test5" time="3.015">
    <failure type="org.testng.internal.thread.ThreadTimeoutException" message="Method practice.CW13_TestNgkey.test5() didn&amp;apos;t finish within the time-out 3000">
      <![CDATA[org.testng.internal.thread.ThreadTimeoutException: Method practice.CW13_TestNgkey.test5() didn't finish within the time-out 3000
at java.base/java.lang.Thread.sleep0(Native Method)
at java.base/java.lang.Thread.sleep(Thread.java:465)
at practice.CW13_TestNgkey.test5(CW13_TestNgkey.java:29)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1589)
... Removed 6 stack frames]]>
    </failure>
  </testcase> <!-- test5 -->
  <testcase classname="practice.CW13_TestNgkey" name="test4" time="0.0">
    <ignored/>
  </testcase> <!-- test4 -->
</testsuite> <!-- MBSTest -->
